module/vision/VisualMesh/src/VisualMesh.cpp

```diff
@@ -169,6 +176,15 @@ namespace module::vision {
                                    100 * double(processed) / double(processed + dropped)));
             processed = 0;
             dropped   = 0;
+
+            double average_fps = (frame_count > 0) ? (1000.0 * frame_count / cumulative_duration_ms) : 0.0;
+
+            log<DEBUG>(fmt::format("Receiving {}/s, Processing {}/s,  Dropping {}/s ({}%), Average FPS: {:.2f}",
+                                   processed + dropped,
+                                   processed,
+                                   dropped,
+                                   100 * double(processed) / double(processed + dropped),
+                                   average_fps));
```

Comment:
shouldn't the debug log above be removed in favour of this one? also this looks like a separate PR.